deploy/start_worker.sh

```diff
@@ -28,4 +28,5 @@ else
 fi
 
 cd /alfalfa
+echo "Python being used is: $(which python)"
```

Comment:
is this correct since the `star_worker.sh` script is not being run with poetry?

Reply:
it is because we set the path to the python in the docker file

Reply:
makes sense